From 3a3effdd81e97bfc4f2698cd1f41b008355b0469 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Tue, 26 May 2020 16:34:45 +0000
Subject: [PATCH] scoped_nss_types.h: Include <certt.h> instead of <nss/certt.h>

Follow-up to commit 701cae5de ("Introduce a function to format a DER-encoded
SPKI for UI").

That is the format used by other files that include <certt.h>, and the
pkg-config configuration already passes -I/path/to/nss to the compiler.

This can potentially fix the non-sysroot build, as the NSS headers might be
installed in e.g. /usr/include/nss3 rather than /usr/include/nss.

Change-Id: I0461a7a7f8c93459f2281ea71c96434a74687ee8
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2214390
Auto-Submit: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Commit-Queue: David Benjamin <davidben@chromium.org>
Reviewed-by: David Benjamin <davidben@chromium.org>
Cr-Commit-Position: refs/heads/master@{#771840}
---

diff --git a/crypto/scoped_nss_types.h b/crypto/scoped_nss_types.h
index 2a3a6e1..0a663ea 100644
--- crypto/scoped_nss_types.h
+++ crypto/scoped_nss_types.h
@@ -5,9 +5,9 @@
 #ifndef CRYPTO_SCOPED_NSS_TYPES_H_
 #define CRYPTO_SCOPED_NSS_TYPES_H_

+#include <certt.h>
 #include <keyhi.h>
 #include <nss.h>
-#include <nss/certt.h>
 #include <pk11pub.h>
 #include <plarena.h>

